packages/sfui/frameworks/react/components/VsfAccordionItem/VsfAccordionItem.tsx

```diff
@@ -16,10 +18,16 @@ export default function VsfAccordionItem({
   const classes = classNames([className, 'vsf-accordion-item']);
   return (
     <details className={classes} open={open} onToggle={onToggle} {...attributes} data-testid="accordion-item">
-      <summary className={`${chevronLeft && 'vsf-accordion-item__header--right'} vsf-accordion-item__header`}>
+      <summary
+        className={classNames(`${chevronLeft && 'vsf-accordion-item__header--right'} vsf-accordion-item__header`)}
+      >
         {slotPrefix || (
           <span
-            className={`${chevronLeft && 'vsf-accordion-item__title--right'} vsf-accordion-item__title`}
+            className={classNames(
```

Comment:
Well, not exactly what I meant. 😅 [Classnames](https://github.com/JedWatson/classnames) provides some handy features, so we don't have to compose classes manually.
```suggestion
            className={classNames({ 
            'vsf-accordion-item__title--right': chevronLeft,
            [`vsf-accordion-item__title vsf-accordion-item__header--${size}`]: true
            })}
```